CMakeLists.txt

```diff
@@ -1,64 +1,75 @@
-cmake_minimum_required(VERSION 3.9.5)
+cmake_minimum_required(VERSION 3.8.0)
```

Comment:
Why was the required version downgraded ? :thinking:

Reply:
I found no features used waranting a higher-than-that version. So this is now more compatible than it was before.